cmd/earthly/app/run.go

```diff
@@ -157,6 +160,18 @@ func (app *EarthlyApp) run(ctx context.Context, args []string) int {
 			"Error: No SetFatalError called appropriately. This should never happen.")
 	}()
 
+	sigChan := make(chan os.Signal, 1)
+	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGINT)
```

Comment:
did you mean to add syscal.SIGINT twice?

Reply:
No. Thanks.